newIDE/app/src/ResourcesList/index.js

```diff
@@ -127,9 +172,22 @@ export default class ResourcesList extends React.Component<Props, State> {
         click: () => this._editName(resource),
       },
       {
-        label: 'Delete',
+        label: 'Remove',
         click: () => this._deleteResource(resource),
       },
+      { type: 'separator' },
+      {
+        label: 'Scan for Images',
```

Comment:
`Scan for images` would be better I think, as "image" is a common name.

Reply:
Well, I'm not sure after looking a bit at some examples of context menu labels in Windows and macOS, you can leave it like this! :)